src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    private static final String[] COLORS = {"red", "green", "blue", "yellow", "black"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
The `Random` instance should be a private class-level variable rather than being instantiated in the `getRandomColor` method. This avoids unnecessary creation of `Random` objects on each method call.